scripts/zfs-helpers.sh

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash
+#
+# This script is designed to facilitate in-tree development and testing
+# by installing symlinks on your system which refer to in-tree helper
+# utilities.  These helper utilities must be installed to in order to
+# exercise all ZFS functionality.  By using symbolic links and keeping
+# the scripts in-tree during development they can be easily modified
+# and those changes tracked.
+#
+# Use the following configuration option to override the installation
+# paths for these scripts.  The correct path is automatically set for
+# most distributions but you can optionally set it for your environment.
+#
+#   --with-mounthelperdir=DIR
+#                           install mount.zfs in dir [[/sbin]]
+#   --with-udevdir=DIR      install udev helpers [default=check]
```

Comment:
This is how it appears in the `./configure --help` output.  It looks like there are a few checks where the m4 is escaped wrong.  `[[ ]]` instead of `@<:@@:>@`.  We should be probably fix that.